src/formatElements.js

```diff
@@ -0,0 +1,88 @@
+export function formatElements(rawElements, version) {
+
+  const majVersion = version.split('.')[0];
+
+  const formattedElements = rawElements
+    .map(({name, type, docs}) => {
+
+      // Format description text
+      const desc = docs.value
+        .split('\n')
+        .map(line => {
+          if (line !== '/**' && line !== ' */') {
+            return line
+              // Remove PHP comment markup
+              .replace(/^\s\*\s{0,1}/g, '')
+
+              // Special/escaped character replacement
+              .replaceAll("&quot;", "\"")
+              .replaceAll(/<([^>]*)>/g, "");
```

Comment:
Actually this was intended to be left untouched, because I find the code example very helpful before choosing how to implement one of the snippets as we don't want to spend time reading it on the file itself or the official webpage.

If we remove all desc after 'Usage example', any suggestions for better research experience?